### ISSUE TITLE
Briya PCS (SC = 0126) - attendance

### ISSUE THREAD

**Issue by gwenrr** [DC-OSSE/LearnDC_v2#144]
Opening a new issue (was #87) because no one other than @benjaminrobinson responded to my question from a few days ago to the thread on the old issue

No attendance data showing on prod, though there is data on staging

![screenshot 2015-05-21 10 52 09](https://cloud.githubusercontent.com/assets/7264714/7751397/b7a5045c-ffa7-11e4-9b89-58f1dc4baf4a.png)

Thank you

**Comment by cmgiven** [comment]
The production data does not have a `year` associated with the unexcused absences. @mepps, is there a workaround that we can implement, short of pushing the staging data live to production?

**Comment by benjaminrobinson** [comment]
Hmm...Weird!
https://github.com/DC-OSSE/LearnDC_v2/blob/master/Export/JSON/school/0126/unexcused_absences.json

**Comment by benjaminrobinson** [comment]
Did something happen in the import to the Social Driver repo?

**Comment by benjaminrobinson** [comment]
Its got a year there too!
https://github.com/socialdriver/learndc-data/blob/master/converted/school/0126/unexcused_absences.json

**Comment by cmgiven** [comment]
Should clarify, I'm just looking at what the API's returning, not what's in your files. Thus marking for Maggie to investigate.

**Comment by benjaminrobinson** [comment]
Ah...I get it!  Good call!  Thanks for the clarification.

**Comment by benjaminrobinson** [comment]
Or rather a bad call, if that joke is not lost on y'all...

**Comment by benjaminrobinson** [comment]
When I'm looking at the production site's API call for the Unexcused Absences part of the attendance exhibit (along with ISA), the data download button call to the API is calling the attendance exhibit instead of the unexcused_absences data...Just another thing OI noticed if that helps!

**Comment by mepps** [comment]
@cmgiven @benjaminrobinson I can definitely fix the data download button. 

It seems to be a problem with that specific import. I can look do more research as to how that happened in the future, but in terms of the data, I was hoping I could import another dataset from the same time but the problem is that that would then be set as "latest" and displayed on staging. I was looking at promoting the April 8th data, but since that has the restructured staff degree files, it will cause an error on the instructional staff exhibit.

What I could do for the short term is promote the previous commit 76bc4d7. It appears the only difference was that we removed the school_ files, but that data does not have the same issues with specific exhibits.

**Comment by cmgiven** [comment]
@benjaminrobinson, does that work for you? That would be the data as of February 25. Here's what it looks like at that point: https://github.com/socialdriver/learndc-data/tree/76bc4d770a856da3d5566febf5835f936f5dbcd3

**Comment by benjaminrobinson** [comment]
@mepps Could you expound a little bit on what you mean by the "school_" files and what the "same issues with specific exhibits" mean?  Gwen isn't in the office today so I'm unsure if I have the authority to make that call but I'd definitely like to know more so I can inform her when we return on Tuesday...

**Comment by mepps** [comment]
I just saw this comment. The school_ files were ones we pulled out previously, I believe they stood in for another exhibit that was later added. How is the LEA review going? @gwenrr this relates to the other issue you opened. I'm hoping we can promote the data soon and that will resolve all these issues.

**Comment by benjaminrobinson** [comment]
LEA review period is over but I'm working on doing the cleaning/preparing of the new data to be combined and then do a few more quality checks.

**Comment by mepps** [comment]
Exciting news! I believe Piotr and I were able to fix these issues, but would love confirmation from @benjaminrobinson @gwenrr @cmgiven

**Comment by cmgiven** [comment]
I belive so. Would still appreciate your eyes @benjaminrobinson @gwenrr.

@mepps, when you get a chance, could you write up a summary of the issue, and in particular how we can improve our process in the future while waiting on long review periods to conclude? Thanks.

**Comment by benjaminrobinson** [comment]
I'll take a look...thanks, @mepps! Gwen is out of the office for a week thing so she won't be as available today but I'm happy to take a look.

**Comment by benjaminrobinson** [comment]
Is this relevant anymore because of my new commush of data?  Just wondering!

**Comment by mepps** [comment]
@benjaminrobinson yes because this is on the live site versus the staging data. It looks like people agree it's looking better but let's wait for @gwenrr's sign off on Monday. Great work on the data!

**Comment by gwenrr** [comment]
Just getting back to this - and yes it looks better since now there is actually data there! 

But here is the issue now. Prod site (including ER) says ISA = 82%; staging site says ISA = 73%. When I download data that are there, it says all student ISA = 82%. The only subgroup that is 73% = males. So maybe a metaphorical crossed wire somewhere? Is this waiting for a new commit?

**Comment by mepps** [comment]
@gwenrr Just to clarify, is the prod data correct on this or staging? Thanks!

**Comment by gwenrr** [comment]
This should not be a complicated answer, but it is. Sigh.

This school enrolls PK3 and PK4 and adults. 82% is the correct ISA rate for PK3 and PK4. This is what's on prod (both in the profile and in the Equity Report).

72% is the rate for adults. This is what's on the profile on staging.

82% is what I'd like it to show on staging and prod - in both profile and ER.

Thank you.

**Comment by mepps** [comment]
![screen shot 2015-06-15 at 9 40 40 am](https://cloud.githubusercontent.com/assets/4406809/8161107/9d87f422-1342-11e5-86ee-0280d3517a0c.png)

![screen shot 2015-06-15 at 9 41 34 am](https://cloud.githubusercontent.com/assets/4406809/8161120/bcd564fe-1342-11e5-93c6-4666e53dcefd.png)

This is fixed on prod and staging.
